### ISSUE TITLE
Add dotted circle (U+25CC)

### ISSUE THREAD

**Issue by tphinney** [googlefonts/science-gothic#325]
Ensure dotted circle glyph is present and can attach marks.
Check ID: com.google.fonts/check/dotted_circle
The dotted circle character (U+25CC) is inserted by shaping engines before mark glyphs which do not have an associated base, especially in the context of broken syllabic clusters.

For fonts containing combining marks, it is recommended that the dotted circle character be included so that these isolated marks can be displayed properly; for fonts supporting complex scripts, this should be considered mandatory.

Additionally, when a dotted circle glyph is present, it should be able to display all marks correctly, meaning that it should contain anchors for all attaching marks.

⚠️ ScienceGothic[YOPQ,slnt,wdth,wght].ttf
⚠️ WARN
No dotted circle glyph present [code: missing-dotted-circle]

**Comment by kateliev** [comment]
@tphinney added. Not yet set up the anchors. Please take a look and tell me if the "design" of the thing will do and will add the necessary anchors :)

**Comment by kateliev** [comment]
@tphinney did a redesign of yesterdays addition :/ and also added anchors. Please review the glyph and let me know what you think.

**Comment by kateliev** [comment]
@tphinney, here are some notes regarding the current design and why I prefer it over the auto-generated version by fontmake:

- **Height:** The height is inspired by what I've observed across various typefaces, particularly Google Sans. In that typeface, the character's height falls between the baseline and a point midway between the x-height and caps height. I chose the height of our small caps based on this observation, as it seemed to fit the best within these parameters.

- **Dot Design:** The design of the "dot" (or circle) follows the approach we used for other punctuation marks in the font—a squarish dot with hyperelliptical corners.

- **Dot Arrangement**: I opted for an even number of dots, each 30 degrees apart, resulting in 12 dots like the face of a clock. This decision was straightforward, though I did notice a wide range of different solutions in other typefaces.

- **Weight Variation:** The weight of the dots varies across the weight axis, with subtle changes in width at the bold masters to make them appear more pronounced.

- **Width Variation**: The width of the dots also varies across the width axis, with changes in inter-dot spacing to create a contraction/expansion effect. This approach seems particularly fitting for our typeface, where width changes dramatically.

![image](https://github.com/user-attachments/assets/8896137d-67b2-4f94-aa32-e7182b28d675)
